src/meta/src/rpc/metrics.rs

```diff
@@ -843,7 +843,7 @@ pub fn start_fragment_info_monitor(
                                 .with_label_values(&[
```

Comment:
we can move the collection to the outer loop of fragment for better performance.

Reply:
Indeed.